pkg/external_repos/redhat_repos.json

```diff
@@ -0,0 +1,37 @@
+[
+    {
+        "name": "Red Hat Ansible Engine 2 for RHEL 8 x86_64 (RPMs)",
+        "url": "https://cdn.redhat.com/content/dist/layered/rhel8/x86_64/ansible/2/os",
```

Comment:
If I update a URL here, it will create the repository in the database, but returns an error creating the repository configuration.

`"ERROR: duplicate key value violates unique constraint \"repo_config_name_deleted_org_id_unique\" (SQLSTATE 23505)`

Do we anticipate needing to handle the case where a to URL to one of these repositories changes, but it's really the same repository?

Reply:
I wrote this thinking we'd want to change everything BUT the url, so this behavior is intentional.  That said, I dont' know what we'll need to do in the future.  So for now i'd say lets keep this as is, and support url changes in the future if needed.